src/test/resources/restapi/rolesmapping_null_array_element.json

```diff
@@ -0,0 +1,11 @@
+{
+  "users": [
+    null
+  ],
+  "backend_roles": [
```

Comment:
add test cases for `backend_roles` and `hosts`

Reply:
Sure, done